core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js

```diff
@@ -205,6 +205,13 @@ oppia.controller('ExplorationEditor', [
 
         StateEditorTutorialFirstTimeService.init(
           data.show_state_editor_tutorial_on_load, $scope.explorationId);
+
+        if (ExplorationRightsService.isPublic()) {
+          // Don't make user wait on stats to begin editing the exploration.
+          StateTopAnswersStatsBackendApiService.fetchStats(
+            $scope.explorationId
+          ).then(StateTopAnswersStatsService.init);
```

Comment:
Don't the new services need to be imported explicitly as dependencies?

Did you test this manually?

Reply:
Done, not yesterday was just concerned w/ getting the changes out. Did today and fixed what seems to have caused all the errors!